Keyboards/KeyboardsBase/KeyboardStyling.swift

```diff
@@ -30,6 +30,21 @@ func styleBtn(btn: UIButton, title: String, radius: CGFloat) {
   btn.layer.masksToBounds = false
   btn.layer.cornerRadius = radius
   btn.setTitle(title, for: .normal)
+  // Displays info button for missing data
+  if title == invalidCommandMsg {
+    if #available(iOS 15.0, *) {
+      var configuration = UIButton.Configuration.plain()
+      configuration.baseForegroundColor = UITraitCollection.current.userInterfaceStyle == .light ? specialKeyColor : keyColor
+      configuration.image = UIImage(systemName: "info.circle.fill")
+      configuration.imagePlacement = .trailing
+      configuration.imagePadding = 3
+      btn.configuration = configuration
+    } else {
+      btn.semanticContentAttribute = .forceLeftToRight // Has to be changed once support for RTL languages is implemented
```

Comment:
Do we want to put this on the line above and have it be some standard we can put places for later? `Note: RTL changes needed` or something like that?